R/get_coverage_wcs.R

```diff
@@ -20,7 +20,8 @@
 #'   - `"omw"`: orthophotomosaic winter images Flanders
 #'   - `"dtm"`: digital terrain model Flanders
 #'   - `"dsm"`: digital surface model Flanders
-#' See [metadata Vlaanderen](https://metadata.vlaanderen.be/metadatacenter/srv/dut/catalog.search#/search?keyword=OGC:WCS) for more information # nolint: line_length_linter.
+#'   - `"dhmv"`: digital elevation model Flanders (contains dtm and dsm data)
+#' See [metadata Vlaanderen](https://metadata.vlaanderen.be/srv/eng/catalog.search#/search?any=WCS) for more information # nolint: line_length_linter.
```

Comment:
The `# nolint: line_length_linter` string appears as-is in the resulting Rd file... I don't know whether roxygen comments have a way to support this, but the current result doesn't look so nice :confused:.

Reply:
I have rearranged that comment.